torch/_prims/__init__.py

```diff
@@ -1828,10 +1829,22 @@ def _bool_return_reduction_meta(inp, dims):
     return _reduction_meta(inp, dims, output_dtype=torch.bool)
 
 
+def _var_reduction_meta(inp, dims, *, correction):
+    if utils.is_complex_dtype(inp.dtype):
+        output_dtype = utils.corresponding_real_dtype(inp.dtype)
+    else:
+        output_dtype = inp.dtype
+    return _reduction_meta(inp, dims, output_dtype=output_dtype)
+
+
 _sum_doc = """
     Computes the sum of elements in the input tensor over the list of dimensions
     specified in the dim argument
     """
+_prod_doc = """
```

Comment:
@ngimel We forgot to update the `_prod_doc` in `prod = _make_reduction_prim` :wink: